db/notifications.go

```diff
@@ -0,0 +1,64 @@
+package db
+
+import (
+	"encoding/json"
+
+	"github.com/flanksource/duty/models"
+	v1 "github.com/flanksource/incident-commander/api/v1"
+	"github.com/google/uuid"
+)
+
+func PersistNotificationFromCRD(obj *v1.Notification) error {
+	uid, err := uuid.Parse(string(obj.GetUID()))
+	if err != nil {
+		return err
+	}
+
+	dbObj := models.Notification{
+		ID:         uid,
+		Events:     obj.Spec.Events,
+		Title:      obj.Spec.Title,
+		Template:   obj.Spec.Template,
+		Filter:     obj.Spec.Filter,
+		Properties: obj.Spec.Properties,
+	}
+
+	if obj.Spec.Person != "" {
+		if uid, err := uuid.Parse(obj.Spec.Person); err == nil {
+			dbObj.PersonID = &uid
+		} else {
+			var person models.Person
+			if err := Gorm.Where("email = ?", obj.Spec.Person).First(&person).Error; err != nil {
+				return err
+			}
+			dbObj.PersonID = &person.ID
+		}
```

Comment:
I think this should move to duty.FindPerson/Team

Reply:
https://github.com/flanksource/duty/pull/277